daemon/daemon.go

```diff
@@ -731,7 +735,8 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 		return nil, err
 	}
 
-	distributionPool := distribution.NewPool()
+	d.downloadManager = xfer.NewLayerDownloadManager(d.layerStore, 5)
```

Comment:
Yes, this should probably be defined in a constant.

The number 5 is arbitrary and just seemed like a sane number of simultaneous transfers. It could be made tunable if people think it's worth the effort.